### DISCUSSION TITLE
service "microanalytics"

### DISCUSSION THREAD

**Discussion by yoan34** [AmauriC/tarteaucitron.js#900] [Ideas]
Bonjour, je cherche une alternative à GA, je suis tombé sur **microanalytics** qui à l'air sympa. Pour l'instant j'analyse notre besoin de gestion de cookie et on risque d'utiliser **tarteaucitorn**.

J'aimerai savoir si je peux ajouter moi-même le service via **tarteaucitron** ou si je dois attendre une feature.

Peut-être existe t-il des solutions similaire à plus simple à GA, je m'y perds un peu avec tous ces services possible et les CMP.

**Comment by AmauriC** [comment]
Bonjour : )

Autant l'ajouter à tarteaucitron pour que tout le monde puise en profiter.

Il me faut le marqueur html, la liste des cookies déposés et le lien de la politique de confidentialité.
Même base que https://github.com/AmauriC/tarteaucitron.js/issues/895

**Reply by yoan34** [reply]
Marqueur HTML
`<script data-host="https://microanalytics.io" data-dnt="false" src="https://microanalytics.io/js/script.js" id="ZwSg9rf6GA" async defer></script>`

Liste des cookies:
ICI je sais pas vraiment quoi mettre, y'a 3 cookies quand je vais sur leur site ("dark_mode", "phpanalytics_session", "XSRF_TOKEN")

Lien vers la politique de confidentialité:
https://microanalytics.io/page/privacy

**Reply by AmauriC** [reply]
Merci, j'ai ouvert un ticket, je m'en occupe rapidement ; )
https://github.com/AmauriC/tarteaucitron.js/issues/902
